tests/ccgbugs/t13902.nim

```diff
@@ -0,0 +1,16 @@
+discard """
+  cmd: "nim c -r --skipParentCfg --skipUserCfg --cpu:i386 --passC:-m32 --passL:-m32 --stacktrace:off --hints:off $file"
+  action: run
+"""
+
+#issue #13902
+when defined(linux):
```

Comment:
It's not specific to Linux. It's specific to 32-bit platforms.

Reply:
but it'd fail on OSX

Reply:
The problem was first discovered on 32-bit Windows, in Azure Pipelines.

A better condition would be `when sizeof(int) == 4: # 32bit`

Reply:
but the test didn't fail on a 32bit linux VM, so the 64 bit cross compilation is needed to see the bugfix on linux; plus it's good to have at least one test showing cross compilation from 64 to 32bit